titus-api/src/main/java/com/netflix/titus/api/model/TokenBucketPolicies.java

```diff
@@ -0,0 +1,37 @@
+package com.netflix.titus.api.model;
+
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.base.Preconditions;
+import com.netflix.titus.common.util.limiter.Limiters;
+import com.netflix.titus.common.util.limiter.tokenbucket.TokenBucket;
+
+/**
+ * Supplementary functions to work with the token bucket policies.
+ */
+public final class TokenBucketPolicies {
+
+    private TokenBucketPolicies() {
+    }
+
+    /**
+     * Create a new token bucket instance from the provided policy.
+     */
+    public static TokenBucket newTokenBucket(String name, TokenBucketPolicy policy) {
+        Preconditions.checkArgument(
+                policy.getRefillPolicy() instanceof FixedIntervalTokenBucketRefillPolicy,
+                "Only FixedIntervalTokenBucketRefillPolicy supported"
+        );
+
+        FixedIntervalTokenBucketRefillPolicy refillPolicy = (FixedIntervalTokenBucketRefillPolicy) policy.getRefillPolicy();
```

Comment:
minor: slight preference for polymorphism rather than `switch (instanceof)` + `cast` here. Consider refactoring into something like `TokenBucketRefillPolicy#newLimiter(capacity)`, in which case the `FixedIntervalTokenBucketRefillPolicy` implementation would returns a `Limiters.createFixedIntervalTokenBucket`.

Reply:
This particular function helps to create a new `TokenBucket` from an abstract policy which is defined somewhere else (hard coded or submitted by a client).